circle.yml

```diff
@@ -20,3 +20,4 @@ deployment:
     branch: master
     commands:
       - NODE_ENV=prod npm run build
+      - npm publish
```

Comment:
@stulees prepublish would actually just run the build. It wouldn't actually publish it. You'd need `npm publish` for that. Order of execution would be `npm publish` > `npm run build` > `gulp build` > Publish to npm.